### PR TITLE
New "how to" create a new blueprint post

### PR DESCRIPTION
**Pull request by wpschaub**
A proposed post that covers the new blueprint architecture and how to build a new blueprint. I suggest we use the blog to create our WHAT, WHY, and HOW documentation for a win:win with the community and WSBC.

Please update the post where needed!

### CODE REVIEW

**Review by wsbctechnicalblog — APPROVED** [review]
Fantastic continuation, let's get this out a.s.a.p. & push it to all EDO engineers. We have no time to waste!

Thank you Willy.

**Review by erwsch1 — APPROVED** [review]

File: content/moving-hundreds-of-pipeline-snowflakes-part9.md [hunk 1]
```diff
@@ -361,5 +361,5 @@ Watch this space for more exiting progress.
 
 ---
 
-> Series Bread Crumbs | [Part 1, TOC](/why-pipelines-part1.html) | [Part 2](/yaml-pipelines-part2.html) | [Part 3](/yaml-pipelines-part3.html) | [Part 4](/yaml-pipelines-part4.html) | [Part 5](/yaml-pipelines-part5.html) | [Part 6](/yaml-pipelines-part6.html) | [Part 7](/yaml-pipelines-part7.html) | [Part 8](/yaml-pipelines-part8.html) | Part 9 |
+> Series Bread Crumbs | [Part 1, TOC](/why-pipelines-part1.html) | [Part 2](/yaml-pipelines-part2.html) | [Part 3](/yaml-pipelines-part3.html) | [Part 4](/yaml-pipelines-part4.html) | [Part 5](/yaml-pipelines-part5.html) | [Part 6](/yaml-pipelines-part6.html) | [Part 7](/yaml-pipelines-part7.html) | [Part 8](/yaml-pipelines-part8.html) | [Part 9](/yaml-pipelines-part9.html) | [Part 10](/yaml-pipelines-part10.html) |
```
Comment (said-akram-wcbbc) [inline content/moving-hundreds-of-pipeline-snowflakes-part9.md:364]: Part 9 should not have a hyperlink

File: content/moving-hundreds-of-pipeline-snowflakes-part10.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
+Title: Pipelines - Meet our second-generation app-type blueprints
+Date: 2021-07-24
+Category: Posts
+Tags: automation, azure-devops, pipelines, x-as-code
+Slug: yaml-pipelines-part10
+Author: Willy-Peter Schaub
+Summary: Our quest for continuous improvement, simplicity, consistency, enablement, and automation has taking us into a realm of Azure Pipelines that amaze even our most critical engineers.
```
Comment (said-akram-wcbbc) [inline content/moving-hundreds-of-pipeline-snowflakes-part10.md:7]: a small misspell: "automation has **taken** us"

**Review by erwsch1 — COMMENTED** [review]
Only one minor correction. Eluded should be alluded

**Pull request merged**